apps/jobs/views.py

```diff
@@ -1440,8 +1441,15 @@ def update_partially_evaluated_submission(request, challenge_pk):
         stderr_content = request.data.get("stderr", "")
         submission_result = request.data.get("result", "")
         metadata = request.data.get("metadata", "")
-        submission = get_submission_model(submission_pk)
-
+        try:
+            submission = get_submission_model(submission_pk)
+        except NotFound:
+            response_data = {
+                "error": "Submission does not exist with submission_id {}".format(
```

Comment:
Change `Submission does not exist with submission_id {}` to `Submission with submission_pk {} does not exist`